spec/fixture_specs/specs_with_syntax_errors.rb

```diff
@@ -1,4 +1,4 @@
-require_relative "../spec_helper"
+require "spec_helper"
```

Comment:
When I committed this, for some reason this `require_relative` was necessary. When I came back, it wasn't. Perhaps something was up with my ENV at the time of the change. In any case I didn't diagnose the root cause.